Graphs/AbstractGraph.js

```diff
@@ -111,7 +111,7 @@ export default class AbstractGraph extends React.Component {
                 let type = this.hoveredDatum.tooltipName || 'default'
                 return this.tooltipContent({ tooltip: this.tooltips[type], accessors: this.accessors[type] })
             } else {
-                return null;
+                return;
```

Comment:
not sure how this will solve the issue and if we need to return undefined for the this.getTooltipContent we might need to do the same for all instances where we defined this.getTooltipContent, see line 87

Reply:
Actually, this error has occurred when any graph has a tooltip and is not completely loaded and a user hovers on it, so in that case browser not able to give coordination (clientX && clientY) to tooltip library and we are returning `null` which is consider as a tooltip content. So when I saw the tooltip library they are just `return`.

On line 87, it is called when there is no tooltip configuration for given graph, so it will not throw any exception.
Please let me know if I am missing anything.

Reply:
Sounds good. Let's push it in